Cannot find module '@color/colors/safe' from 'table-test.js'

The tests are presently failing with the error: `Cannot find module '@color/colors/safe' from 'table-test.js'`.

This bug should not affect production installations and is only induced when running this test.

This bug was introduced in #263 and not immediately noticed because of #271.

--- a/test/table-test.js
+++ b/test/table-test.js
@@ -1,6 +1,6 @@
 describe('@api Table ', function () {
   const Table = require('..');
-  const colors = require('@color/colors/safe');
+  const colors = require('@colors/colors/safe');
 
   it('wordWrap with colored text', function () {
     let table = new Table({ style: { border: [], head: [] }, wordWrap: true, colWidths: [7, 9] });

